src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19955.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Drawing;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+using OpenQA.Selenium.Interactions;
+using NUnit.Framework.Legacy;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue19955: _IssuesUITest
+{
+    public Issue19955(TestDevice device) : base(device) { }
+
+    public override string Issue => "Navigating Back to FlyoutPage Renders Blank Page";
+
+    [Test]
+	[Category(UITestCategories.FlyoutPage)]
+	public void NavigatingBackToFlyoutPageRendersBlankPage()
+    {
+		App.WaitForElement("NavigateToSecondPageButton");
+        App.Tap("NavigateToSecondPageButton");
+        App.WaitForElement("NavigateBackToFirstPageButton");
+        App.Tap("NavigateBackToFirstPageButton");
+		App.WaitForElement("NavigateToSecondPageButton");
```

Comment:
How does `WaitForElement` test for visibility?

Reply:
It looks like if the opacity is zero then appium can't find it. 

I was curious about this as well :-) 
If I remove my fix, the test fails, if I put the fix back, the test passes.

Reply:
If the element is not visible on the screen, appium can't find it. So, it's valid. Just could be nice to include a comment here.